src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/TypeAccessibilityCodeActionProvider.cs

```diff
@@ -27,6 +30,8 @@ internal class TypeAccessibilityCodeActionProvider : CSharpCodeActionProvider
             "IDE1007"
         };
 
+        private static readonly Regex AddUsingVSCodeAction = new Regex("^using .+;$", RegexOptions.Compiled, TimeSpan.FromSeconds(1));
```

Comment:
why do we have two copies of the regex (both subtly different?)

Reply:
It was due to how the edit differed from the title (edit could have leading/trailing whitespace. Updated to use a single uniform regex, trimming whitespace for the edit.